### ISSUE TITLE
Avoid PersistentBehaviorSpec$.MODULE$.config() in Java

### ISSUE THREAD

**Issue by ktoso** [akka/akka#24692]
In

```
 +  public static final TestKitJunitResource testKit = new TestKitJunitResource(PersistentBehaviorSpec$.MODULE$.config()); 
```

Avoiding the MODULE yields with

```
error] /Users/ktoso/code/akka/akka-persistence-typed/src/test/java/akka/persistence/typed/javadsl/PersistentActorTest.java:29:1: non-static method config() cannot be referenced from a static context
[error]   public static final TestKitJunitResource testKit = new TestKitJunitResource(PersistentBehaviorSpec.config());

```

**Comment by patriknw** [comment]
That's surprising. It's an ordinary class + object

**Comment by johanandren** [comment]
Fixed by #24753
